.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ".eslintrc-node.json",
```

Comment:
Please double-check all JSON files use tabs to indent.

Reply:
Done. Many thanks

Reply:
Some files are still not indented via tabs, e.g. `.eslintrc-browser.json`.

Reply:
I've added another comment about that so I'm resolving this thread.